common/types/account_storage.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+//go:generate scalegen
+
+// StorageItem represents a single item of account storage.
+type StorageItem struct {
+	// not actually a hash, just 32 bytes
+	Key   Hash32
```

Comment:
How about using a 32 array then and dropping the comment? :)
```suggestion
	Key   [32]byte
```